.github/workflows/test.yml

```diff
@@ -29,7 +29,8 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           sudo apt-get -y update
-          sudo apt-get -y install conntrack
+          sudo apt-get -y upgrade
+          sudo apt-get -y install conntrack dbus-x11
```

Comment:
```
Preparing to unpack .../dbus-x11_1.12.16-2ubuntu2.1_amd64.deb ...
Unpacking dbus-x11 (1.12.16-2ubuntu2.1) ...
Setting up conntrack (1:1.4.5-2) ...
Setting up dbus-x11 (1.12.16-2ubuntu2.1) ...
Processing triggers for man-db (2.9.1-1) ...
time="2021-02-18T20:15:23Z" level=warning msg="Failed to add podman to systemd sandbox cgroup: dbus: authentication failed"
Version:      3.0.0
API Version:  3.0.0
Go Version:   go1.15.2
Built:        Thu Jan  1 00:00:00 1970
OS/Arch:      linux/amd64
```

Reply:
Also further down in the logs after installing the minikube binary:

```
! The 'none' driver is designed for experts who need to integrate with an existing VM
* Configuring local host environment ...
* Most users should use the newer 'docker' driver instead, which does not require root!
* For more information, see: https://minikube.sigs.k8s.io/docs/reference/drivers/none/
* 
! kubectl and minikube configuration will be stored in /home/runner
! To use kubectl or minikube commands as your own user, you may need to relocate them. For example, to overwrite your own settings, run:
* 
  - sudo mv /home/runner/.kube /home/runner/.minikube $HOME
  - sudo chown -R $USER $HOME/.kube $HOME/.minikube
* 
* This can also be done automatically by setting the env var CHANGE_MINIKUBE_NONE_USER=true
* Verifying Kubernetes components...
* Enabled addons: storage-provisioner, default-storageclass
* Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
X Exiting due to ENV_DRIVER_CONFLICT: 'none' driver does not support 'minikube docker-env' command
```

I'm surprised that CI isn't erroring looking at those logs. It's been a while since I've had to play around with minikube so it's possible those are just warning messages. We may need to add some bash directives to stop running actions when a check fails.